src/components/Inputs/Input/index.tsx

```diff
@@ -0,0 +1,111 @@
+import styles from './Input.module.scss';
+import { useState, InputHTMLAttributes, MouseEvent } from 'react';
+import Image from 'next/image';
+import {
+  FieldErrors,
+  FieldValues,
+  UseFormRegisterReturn,
+} from 'react-hook-form';
+import eyeOnImg from '@/svgs/eye-on.svg';
+import eyeOffImg from '@/svgs/eye-off.svg';
+import searchIcon from '@/svgs/search.svg';
+import calendarIcon from '@/svgs/calendar.svg';
+
+type IconType = 'search';
+
+const ICON_IMG = {
+  search: {
+    src: searchIcon,
+    alt: 'searchIcon',
+  },
+};
```

Comment:
```suggestion
const ICON_IMG = {
  search: {
    src: searchIcon,
    alt: 'searchIcon',
  },
};
```
이렇게 따로 빼서 사용하는 이유가 있나요??
뭔가 이미지를 깔끔하게 관리하시는 의도로 만든 것 같은데 패스워드 눈 아이콘도 넣을 수는 없을까요?!

그리고 src에 넣을 때 한번 import 거치는 게 아니라 바로 경로를 넣어서 코드를 줄이는 건 어떠신가요??
```js
search: {
    src: '@/svgs/search.svg',
    alt: 'searchIcon',
  },
```

Reply:
반영했습니다!